packages/plugins/integration-make-com/src/lib/make-com.middleware.ts

```
@@ -0,0 +1,121 @@
+									isArchived: false,
+									integration: {
+										isActive: true,
+										isArchived: false
+									}
+								},
+								relations: {
+									settings: true
+								}
+							});
+
+							if (fromDb && fromDb.settings) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)